### PR TITLE
Support for Shipping Taxes

### PR DESCRIPTION
**Pull request by rithiksharma100**

### CODE REVIEW

File: includes/helpers/class-simpl-wc-helper.php [hunk 1]
```diff
@@ -56,15 +56,6 @@ static protected function set_data_from_cart( &$order ) {
         WC()->checkout->create_order_coupon_lines( $order, WC()->cart );
     }
 
-    static protected function update_data_from_cart( &$order ) {
```
Comment (Chhekur) [inline includes/helpers/class-simpl-wc-helper.php:59]: now we don't need this is it, why?

Comment (rithiksharma100) [reply includes/helpers/class-simpl-wc-helper.php:59]: yes, because `$order->calculate_totals()` does all this

**Review by abhaymaniyar — DISMISSED** [review]

**Review by Chhekur — APPROVED** [review]
LGTM, it won't break anything for all the merchants who are live on older versions?

**Comment by rithiksharma100** [comment]
> LGTM, it won't break anything for all the merchants who are live on older versions?

No it won't I've tested that scenario as well

**Pull request merged**